hles/transformation/src/test/scala/org/broadinstitute/monster/dap/HLESurveyTransformationPipelineValidationSpec.scala

```diff
@@ -0,0 +1,74 @@
+package org.broadinstitute.monster.dap
+
+import com.spotify.scio.testing.PipelineSpec
+import org.apache.beam.sdk.options.{PipelineOptions, PipelineOptionsFactory}
+import org.broadinstitute.monster.common.PipelineCoders
+import org.broadinstitute.monster.common.msg.JsonParser
+import org.broadinstitute.monster.dap.PostProcess.errCount
+import org.scalatest
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import better.files.File
+import org.apache.beam.sdk.metrics.MetricName
+
+import scala.io.Source
+
+class HLESurveyTransformationPipelineValidationSpec
+    extends AnyFlatSpec
+    with Matchers
+    with PipelineSpec
+    with PipelineCoders
+    with BeforeAndAfterAll {
+  behavior of "HLESurveyTransformationPipeline"
+
+  val opts: PipelineOptions = PipelineOptionsFactory.fromArgs("--runner=DirectRunner").create()
+
+  // set up a pipeline output directory that we can then clean up
+  val outputDir = File.newTemporaryDirectory()
+  override def afterAll(): Unit = outputDir.delete()
+
+  def pipelineTest(
+    inputPath: String,
+    expectedErrorMessage: String,
+    count: Int = 1
+  ): scalatest.Assertion = {
+    val result = runWithRealContext(opts)(ctx =>
```

Comment:
🌱 I like this pattern (i.e., feeding fixture data from the test resources dir into the pipeline, rather than making live calls against the API). I think we should convert our ITs over to this once approach once this is merged.